prod_gen3_test/domu.xml

```diff
@@ -8,9 +8,11 @@
 
     <default remote="github" sync-j="10" sync-c="true" />
 
-    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="krogoth" revision="95cb48ba09bc7e55fd549817e3e26723409e68d5" />
-    <project remote="yoctoproject" name="poky" path="poky" upstream="krogoth" revision="cca8dd15c8096626052f6d8d25ff1e9a606104a3" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="krogoth" revision="7acbf5e2f99c59478adbc73c6a40d314589a3009" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="krogoth" revision="krogoth" />
     <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="krogoth" revision="55c8a76da5dc099a7bc3838495c672140cedb78e" />
+    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="morty" />
```

Comment:
You are not only changing commits for project, but  also you are adding new projects to the manifest. I think, it should be covered in the commit message. 
Also, I would like to see more elaborate commit messages. I see that you are changing commits, but I don't know why are you doing this.

Reply:
Done